dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.eeCompatible-9.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.eeCompatible-9.0
+visibility=private
+singleton=true
+Subsystem-Version: 9.0.0
+-bundles=com.ibm.ws.javaee.version
+kind=ga
```

Comment:
should be `kind=noship` and `edition=full`

Reply:
done